### ISSUE TITLE
Handling a case when the app is already running an user tries to open 2nd instance

### ISSUE THREAD

**Issue by betterversion** [filahf/budgie-stream#20]
Currently it crashes with such js error. Suggest to warn with some nice info message/open current instance of an app instead.

![image](https://user-images.githubusercontent.com/23748065/97240921-84d8f180-17bd-11eb-9eee-fa27abf86a1a.png)

**Comment by filahf** [comment]
Good catch, I'll implement some error handling for that. Thank you for the feedback!

**Comment by Shamatix** [comment]
This is still an issue.
The error says something is already in use, I expect port 5000?
Well I dont have the port in use, the problem started occuring after installing Bungie stream on a second PC.
I closed Bungie stream on my first PC, but I still get this error on my second PC, even after shutting down my first PC?

Further more when I am finally connected to my sonos it doesnt play any sound what so ever.
